src/python/pants/backend/python/rules/coverage.py

```diff
@@ -0,0 +1,279 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import configparser
+import json
+from dataclasses import dataclass
+from io import StringIO
+from typing import Dict, Tuple
+import itertools
+
+import pkg_resources
```

Comment:
The BUILD for this target should now explicitly declare a dep on `3rdparty/python:setuptools`.

Reply:
Bump on this.